Engine/EditorUI/Panel/InspectorSubpanel/PanelGameObject.cpp

```diff
@@ -50,7 +50,7 @@ void PanelGameObject::Render(GameObject* game_object)
 		return;
 	}
 
-	ImGui::PushID(game_object->UUID);
+	ImGui::PushID((int)game_object->UUID);
```

Comment:
I saw you use in some cases (int) in other static_cast<int>() I think it would be better to do static_cast<int>() to standardize the casting in the whole engine